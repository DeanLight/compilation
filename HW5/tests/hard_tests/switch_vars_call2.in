int foo6()
{
	print("\nHello\n");
}

void main ()
{
	int y=1;
	int x =4;
	switch(x)
	{
		default: int vv0;
		case 0: break;
		case 1:
		case 2: int vv1;
		case 3: int vv2;
				printi(vv2);
		case 4: int vv3=3;
				printi(vv3);
		case 5: int vv4=4;
				printi(vv4);
				printi(y);
		case 6: foo6();
		case 7: break;
	};
}