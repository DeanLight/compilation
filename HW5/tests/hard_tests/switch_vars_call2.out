Loaded: ./exceptions.s
341
Hello
